src/aap_eda/api/serializers/activation.py

```diff
@@ -591,8 +602,8 @@ def update(
         self, instance: models.Activation, validated_data: dict
     ) -> None:
         update_fields = []
-        eda_credentials = []
-        event_streams = []
+        eda_credentials = None
```

Comment:
@bzwei You can have more than 1 EventStreams and more than 1 Credential that goes with it.

Reply:
validated_data["event_streams"] and validated_data["eda_credentials"] are already collections. eda_credentials here will be either None or a collection.